client/src/helpers/keycloakHelpers.ts

```diff
@@ -0,0 +1,27 @@
+// check if keycloak has been configured
+export const keycloakEnabled = (config: any) => {
```

Comment:
I like where helper will become some kinda facade that will be available as instance with initialized keycloak and information about enablement mounted user etc. This might not be the nicest thing to do for react so leaving that up to yourself.